source/configure/file-storage-configuration-settings.rst

```diff
@@ -494,4 +494,40 @@ Initial font
 |                                                               | - Environment variable: ``MM_FILESETTINGS_INITIALFONT``                        |
 | A string with the font file name. Default is                  |                                                                                |
 | **nunito-bold.ttf**.                                          |                                                                                |
-+---------------------------------------------------------------+--------------------------------------------------------------------------------+
\ No newline at end of file
++---------------------------------------------------------------+--------------------------------------------------------------------------------+
+
+.. config:setting:: file-amazons3requesttimeoutmilliseconds
+  :displayname: Amazon S3 request timeout (File Storage)
+  :systemconsole: N/A
+  :configjson: .FileSettings.AmazonS3RequestTimeoutMilliseconds
+  :environment: MM_FILESETTINGS_AMAZONS3REQUESTTIMEOUTMILLISECONDS
+  :description: Amount of time, in milliseconds, before requests to Amazon S3 time out. Default value is 30000 (30 seconds).
+
+Amazon S3 request timeout 
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
++---------------------------------------------------------------+-----------------------------------------------------------------------------------------+
+| The amount of time, in milliseconds, before requests to       | - System Config path: N/A                                                               |
+| Amazon S3 storage time out.                                   | - ``config.json`` setting: ``".FileSettings.AmazonS3RequestTimeoutMilliseconds: 30000`` |
+|                                                               | - Environment variable: ``MM_FILESETTINGS_AMAZONS3REQUESTTIMEOUTMILLISECONDS``          |
+| Default is 30000 (30 seconds).                                |                                                                                         |
++---------------------------------------------------------------+-----------------------------------------------------------------------------------------+
+
+.. config:setting:: file-maxpayloadsize
+  :displayname: Maximum payload size (File Storage)
+  :systemconsole: N/A
+  :configjson: .FileSettings.MaximumPayloadSizeBytes
+  :environment: MM_FILESETTINGS_MAXIMUMPAYLOADSIZEBYTES
+  :description: The payload size, in bytes, of API endpoints passing in arrays. Default is 300000.
```

Comment:
```suggestion
  :description: The maximum payload size in bytes for all APIs except APIs that receive a file as an input, for example, the upload attachment API or the API to upload a custom emoji. Default is 300000.
```

Reply:
@cwarnermm I've provided a text suggestion. Its not applicable to only arrays, but rather to all APIs except those that receive a file.